Makefile

```diff
@@ -37,11 +37,8 @@ gen-mocks:
 run-db:
 	@docker run \
 		-d \
-		-v `pwd`/db:/docker-entrypoint-initdb.d/ \
+		-v `pwd`/db:/db \
```

Comment:
Maybe it is better to add a new command with MongoDB support rather than rewrite the old one.

Reply:
I'm not sure about that. What happens if we leave the old run-db command in Makefile and run it with changed main.go file that doesnt support the sql db?

Reply:
It will just return an error that it cannot connect to MongoDB.
You can separate these commands by name. For example, the first can be run-psql, and the other can be run-Mongo.
I was just confused by the fact that you left the SQL implementation of the repo. But removed it from Makefile.
But this is not a matter of principle at all.